app/views/demographic_surveys/new.html.slim

```diff
@@ -8,9 +8,8 @@
     = raw t ".banner"
 
   p.static-page-desc
-    | If you have any questions, please contact us
-    span< = link_to "here", new_contact_form_path
-    |!
+    | If you have any questions, please contact us at <a href="team@calblueprint.org">team@calblueprint.org</a>.
+    | Good luck!
```

Comment:
oh can you remove the "good luck" part on this page? since they have already submitted at this point